root_script: network_maker.sbat

/sw/arcts/centos7/python3.8-anaconda/2021.05/bin/python

spinning up at: Tue Mar 22 23:02:14 EDT 2022
Job submitted by gl-login2.arc-ts.umich.edu
Job ID 33848398
Running on gl3312
Running in /home/casonk/pushshift_python/Great_Lakes_HPC/sbats 

Traceback (most recent call last):
  File "network_maker.py", line 7, in <module>
    os.chdir('/home/casonk/pushshift_python/Python')
FileNotFoundError: [Errno 2] No such file or directory: '/home/casonk/pushshift_python/Python'
backing up data to turbo buffer
source_csvz := /scratch/mmani_root/mmani0/shared_data/hot/csv_networkz/
target_csvz := /nfs/turbo/flnt-mmani/Reddit_Communities/buffer/csv_networkz/

0 files to copy

target_csvz directory purged

copying *.csv*
cp: cannot stat ‘/scratch/mmani_root/mmani0/shared_data/hot/csv_networkz/*.csv*’: No such file or directory

1 of 0 files copied

source_pklz := /scratch/mmani_root/mmani0/shared_data/hot/pkl_networkz/
target_pklz := /nfs/turbo/flnt-mmani/Reddit_Communities/buffer/pkl_networkz/

0 files to copy

target directory purged

copying *.pkl*
cp: cannot stat ‘/scratch/mmani_root/mmani0/shared_data/hot/pkl_networkz/*.pkl*’: No such file or directory

1 of 0 files copied

spinning down at: Tue Mar 22 23:02:14 EDT 2022
